### PR TITLE
Bug 1799698 - part 3: Link focus to the `main` branch of the monorepo

### PR DESCRIPTION
**Pull request by JohanLorenzo**
r? @gabrielluong

### CODE REVIEW

File: taskcluster/ci/update-project/kind.yml [hunk 1]
```diff
@@ -30,18 +30,13 @@ job-defaults:
 jobs:
     mozilla-mobile/android-components:
         repo-prefix: ac
-        pr-target: main
```
Comment (JohanLorenzo) [inline taskcluster/ci/update-project/kind.yml:33]: The default value is already set a few lines above at

https://github.com/mozilla-l10n/android-l10n-tooling/blob/88bf54d10b09b3ed330923edc1989f825a996ae8/taskcluster/ci/update-project/kind.yml#L20

**Comment by JohanLorenzo** [comment]
Jobs are currently red because we haven't merged focus to https://github.com/mozilla-mobile/firefox-android yet. It'll happen tomorrow.

**Review by gbrownmozilla — APPROVED** [review]

**Review by gabrielluong — APPROVED** [review]
Ah, good catch! I forgot about this in our TODOs

**Comment by gabrielluong** [comment]
Can we also remove https://github.com/mozilla-l10n/android-l10n-tooling/blob/0ca326e779646c3787a33d806eb13ed58460dd7a/taskcluster/ci/config.yml#L50 while we are at it?

**Comment by JohanLorenzo** [comment]
> Can we also remove [...]

Done!

**Pull request merged**